pykanto/signal/cluster.py

```diff
@@ -75,7 +74,6 @@ def umap_reduce(
         reducer = cumlUMAP(
             n_neighbors=n_neighbors,
             n_components=n_components,
-            min_dist=min_dist,
```

Comment:
These should be passed as part of the kwargs dict for the same reason as comment in L52-55